Client.php

```diff
@@ -102,22 +101,20 @@ public function __construct ( Socket\Client $client, $endPoint = '/',
     /**
      * Connect, i.e. open the connection and do handshake.
      *
-     * @access  public
      * @return  void
      */
-    public function connect ( ) {
```

Comment:
not correct. You are right.

*Hari K T*

You can ring me : +91 9388 75 8821

http://harikt.com , https://github.com/harikt ,
http://www.linkedin.com/in/harikt , http://www.xing.com/profile/Hari_KT

Skype  : kthari85
Twitter : harikt

On Tue, Apr 7, 2015 at 12:36 PM, Ivan Enderlin <notifications@github.com>
wrote:

> In Client.php
> <https://github.com/hoaproject/Websocket/pull/34#discussion_r27855311>:
>
> >       * @return  void
> >       */
> > -    public function connect ( ) {
>
> But connect( ) is not valid regarding PSR-1 or -2, isn't it?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/hoaproject/Websocket/pull/34/files#r27855311>.
>